js/join.js

```diff
@@ -182,18 +182,33 @@ function arePersonalDetailsValid() {
     country.value.trim() != ''
   );
 }
-
 function introPageValidator() {
+  let countIntroduction = 0;
```

Comment:
You are correcting textarea of the intro page only not for other page

Reply:
Because this function returns a boolean value and like on line `263` it used in a if condition, where it decides that whether in the button , the button-disabled class  should be remove  and button-enabled class should be add . It depends on the introPageValidator function. And this fiunction only works on introPageValidator . Because it only tells about the next Button condition